src/python/CMakeLists.txt

```diff
@@ -363,6 +363,17 @@ if(DEBUG_TRACES)
   set(GUDHI_PYTHON_EXTRA_COMPILE_ARGS "${GUDHI_PYTHON_EXTRA_COMPILE_ARGS}'-DDEBUG_TRACES', ")
 endif(DEBUG_TRACES)
 
+if(CMAKE_BUILD_TYPE MATCHES Debug)
+  set(GUDHI_COMPILATION_FLAGS_STRING ${CMAKE_CXX_FLAGS_DEBUG})
+else()
+  set(GUDHI_COMPILATION_FLAGS_STRING ${CMAKE_CXX_FLAGS_RELEASE})
+endif()
+# Compilation flags are in a string. Transform it to a list and append them to GUDHI_PYTHON_EXTRA_COMPILE_ARGS
+string(REPLACE " " ";" GUDHI_COMPILATION_FLAGS_LIST ${GUDHI_COMPILATION_FLAGS_STRING})
```

Comment:
I assume it breaks if the flags contain `-I/some/dir with/a space/`? It is probably already broken elsewhere in our build stuff though.